secure_config.py

```
@@ -195,7 +195,11 @@ def decrypt_value(value: str) -> str:
         fernet = Fernet(key)
         decrypted: bytes = fernet.decrypt(encrypted_data.encode())
         return decrypted.decode()
-    except (InvalidToken, ValueError, TypeError) as e:
+    except (
+        InvalidToken if ENCRYPTION_AVAILABLE else InvalidTokenPlaceholder,
```

Ruff (0.13.3)

`except` handlers should only be exception classes or tuples of exception classes

(B030)